doc/admin/pool_operations.md

```diff
@@ -8,12 +8,12 @@ management interface.
 ## Pool Creation/Destroy
 
 A DAOS pool can be created and destroyed through the DAOS management API
-(see daos_mgmt.h). DAOS also provides a utility called dmg to manage
+(see daos_mgmt.h). DAOS also provides a utility called dmg_old to manage
 storage pools from the command line.
 
 **To create a pool:**
 ```
-$ dmg create --size=xxG --nvme=yyT
+$ dmg_old create --size=xxG --nvme=yyT
```

Comment:
Shouldn't these examples use the new tool?

Reply:
yes probably but not within the remit site of this PR IMO, that's a separate documentation change task https://jira.hpdd.intel.com/browse/DAOS-3565